roles/prometheus-postgres-exporter/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: restart prometheus-postgres-exporter
+  become: true
+  service:
+    daemon_reload: true
```

Comment:
Ah, it seems it doesn't so systemd would be needed to use this.

Talking with volcyy, it was needed to ensure that the service picked up the new config we set in the previous step.

Reply:
For the systemd role, may be personal preference, but it may be nicer to have a `systemd` role that this role depends on, in that role just have a handler that does `daemon_reload`, otherwise we will start duplicating this across roles a lot.

Reply:
I've removed daemon_reload in this commit https://github.com/python-
discord/infra/pull/58/commits/9a0e0321b55fd3ef323de7dfffeea3734c14bde1

I don't quite understand your suggestion though volcyy, are you saying to add a new roll that just does the daemon_reload and the add that as a dependency in the meta of this role?